_about/FAQ.md

```diff
@@ -4,21 +4,24 @@
 
 The HashiCorp Terraform vSphere provider team is :
 
-* Phil Sautter, Product Manager - [@redeux](https://github.com/redeux)
+* Tejitha Raju, Product Manager - [@tejavar](https://github.com/tejavar)
 * Alex Pilon, Engineer - [@appilon](https://github.com/appilon)
-* Kyriakos Oikonomakos, Engineer - [@koikonom](https://github.com/koikonom)
-* Aareet Mahadevan, Engineering Manager - [@aareet](https://github.com/aareet)
+* Brandy Jackson, Engineering Manager - [@ibrandyjackson](https://github.com/ibrandyjackson)
 
 ### Why isn’t my PR merged yet?
 
-We do our best to focus on the contributions that provide the greatest value to the most community members.
+Due to the volume of issues and new pull requests we receive, we are unable to give each one the full attention that we would like. We do our best to focus on the contributions that provide the greatest value to the most community members. 
 
 ### How do you decide what gets merged for each release?
 
-The number one factor we look at when deciding what issues to look at are your 👍 [reactions](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue/PR description as these can be easily discovered. Comments that further explain desired use cases or poor user experience are also heavily factored. The items with the most support are always on our radar, and we do our best to keep the community updated on their status and potential timelines.
+The number one factor we look at when deciding what issues to look at are your 👍 [reactions](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue/PR description as these can be easily discovered. Comments that further explain desired use cases or poor user experience are also heavily factored. The more edge-case the issue the less value is added. The items with the most community support are always on our radar, and we do our best to keep the community updated on their status and potential timelines.
```

Comment:
```suggestion
The number one factor we look at when deciding what issues to look at are your 👍 [reactions](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue/PR description as these can be easily discovered. Comments that further explain desired use cases or poor user experience are also heavily factored. The items with the most support are always on our radar, and we do our best to keep the community updated on their status and potential timelines, however we will focus on features that will benefit a majority of users.
```

Reply:
since i had already moved forward with these changes, I could not commit via suggestion - I committed the changes myself - thanks for the review/suggestion 👍